frontend/pages/hosts/details/HostDetailsPage/modals/SelectQueryModal/_styles.scss

```diff
@@ -1,76 +1,34 @@
 .select-query-modal {
-  &__no-queries {
-    .info {
-      &__header {
-        margin: $pad-medium 0 $pad-medium 0;
-      }
-      &__data {
-        margin: 0 0 $pad-large 0;
-      }
-    }
-  }
+  min-height: 400px;
+  height: 80%;
+  overflow: hidden;
 
-  &__modal {
-    .info {
-      &__header {
-        display: block;
-        color: $core-fleet-black;
-        font-weight: $bold;
-        font-size: $x-small;
-        text-align: center;
-      }
-
-      &__data {
-        display: block;
-        color: $core-fleet-black;
-        font-weight: normal;
-        font-size: $x-small;
-        text-align: center;
-      }
-    }
-    .modal-cta-wrap {
-      justify-content: center;
-    }
+  .modal__content {
+    display: flex;
+    flex-direction: column;
+    gap: $pad-large;
+    height: 80%;
+    overflow: scroll;
```

Comment:
We'll want `auto` here to have scrollbars just when there's overflow

Reply:
To avoid this: ![Screenshot 2024-04-03 at 10 50 01 PM](https://github.com/fleetdm/fleet/assets/61553566/8356dccc-18f9-4eff-b837-b6e2ffaa56b8)

Reply:
sweet, thank you!